tests/Unit/DataStructures/VectorImplTestHelper.hpp

```diff
@@ -366,5 +376,338 @@ void vector_test_math_after_move(
                           (VectorType{size, sum_generated_values}));
   }
 }
+
+using Bound = std::array<double, 2>;
```

Comment:
Used in `Test_DataVector.cpp`. Allows for the distribution bounds to be declared in a more expressive way. Do you have a suggestion for where in this file it should be instead?

Reply:
I prefer it in the test helper generally, as all of the various vector tests (e.g. `Test_ComplexDataVector` etc) will also need to define distribution bounds; is it better if I just document its intent with a code comment? Ultimately, as it is only a single line, it would be fine if you were to insist on duplicating it in each test.

Reply:
Since the alias is in a namespace I'm fine with it being left and documented as you did :)